sortinghat/core/schema.py

```diff
@@ -933,10 +933,11 @@ def resolve_job(self, info, job_id):
                 for uuid, matches in job.result['results'].items()
             ]
         elif (job.result) and (job_type == 'unify'):
-            errors = job.result['errors']
             result = [
                 UnifyResultType(merged=job.result['results'])
             ]
+        elif status == 'failed':
```

Comment:
`errors` variable should be left in `unify` block to return the know errors. When an unexpected error happens, we can use this part of the code. Also, we need a unit test for this case :).

Reply:
I've left the `errors` variable in the `unify` block and added a test for this case.